src/config.js

```diff
@@ -33,7 +33,7 @@ export const codeSidebarWidth = {
   open: 480,
 };
 
-export const workFlowStatuses = ['success', 'failed', 'skipped'];
+export const workFlowStatuses = ['success', 'failed'];
```

Comment:
please ignore this part, I'll address this after merging this PR to the draw nodes